client/js/verification-portal/pregnancy-resource-center/form.js

```diff
@@ -82,6 +85,7 @@ class VerificationPortalForm extends Component {
 							/>
 						</div>
 					</div>
+					<Spacer height="15px" />
```

Comment:
This space is 64px

<img width="296" alt="screen shot 2018-07-03 at 2 38 32 pm" src="https://user-images.githubusercontent.com/6378435/42241114-110244b6-7ecf-11e8-8247-c7646836b99f.png">